gv.c

```diff
@@ -2819,11 +2823,20 @@ Perl_gv_check(pTHX_ HV *stash)
                 CopFILEGV(PL_curcop)
                     = gv_fetchfile_flags(file, HEK_LEN(GvFILE_HEK(gv)), 0);
 #endif
-                Perl_warner(aTHX_ packWARN(WARN_ONCE),
-                        "Name \"%" HEKf "::%" HEKf
-                        "\" used only once: possible typo",
-                            HEKfARG(HvNAME_HEK(stash)),
-                            HEKfARG(GvNAME_HEK(gv)));
+                if (GvONCE_FATAL(gv)) {
+                    Perl_fatal_warner(aTHX_ packWARN(WARN_ONCE),
```

Comment:
Does this need to use the full `Perl_...(aTHX_...)` version of a function call? Is the macro generated in `embed.h` visible here, so this could be just `fatal_warner(packWARN...)`?